forte/data/ontology/top.py

```diff
@@ -412,7 +341,10 @@ def add_member(self, member: Entry):
                 f"The members of {type(self)} should be "
                 f"instances of {self.MemberType}, but got {type(member)}"
             )
-        self.pack.get_entry_raw(self.tid)[MEMBER_TID_INDEX].append(member.tid)
+        if self.members is None:
```

Comment:
When do we hit this condition? I thought after `__init__` is called, `self.members` should no longer be `None`.

Reply:
That's right. I will remove it